appveyor.yml

```diff
@@ -0,0 +1,54 @@
+version: 4.0.{build}
+image: Visual Studio 2015
+
+
+clone_depth: 1
+
+
+environment:
+  matrix:
+  - PlatformToolset: v140
```

Comment:
@donho Is windows XP still a relevant build target? In this case another platform toolset could be added, e.g. v140_xp